config/default.config

```diff
@@ -22,7 +22,7 @@ params {
     gatk_version       = '4.4.0.0'
     pipeval_version    = '5.0.0-rc.3'
     samtools_version   = '1.20'
-    stablelift_version = 'dev'    // FIXME
+    stablelift_version = '1.0.0'
```

Comment:
Just FYI, this image won't exist until we create the `v1.0.0` git tag. That's not a problem, and I don't know any way around it, but it is a frustration point.

Reply:
Aye this is the unfortunate side-effect with having both bundled in the same repo and tag creation triggering the image build